src/pudl/analysis/plant_parts_eia.py

```diff
@@ -790,7 +790,9 @@ def add_additional_cols(self, plant_parts_eia, plants_eia860, utils_eia860):
                 ],
             )
             .merge(
-                utils_eia860[["utility_id_eia", "utility_id_pudl"]].drop_duplicates(),
+                utils_eia860[
+                    ["utility_id_eia", "utility_id_pudl", "utility_name_eia"]
```

Comment:
I added `utility_name_eia` as a column in the plant parts list so that it can be used for matching.